src/types/colorTable.cpp

```diff
@@ -286,685 +286,274 @@ static constexpr std::array<til::color, 256> standardXterm256ColorTable{
-    std::pair{ "tan2"sv, til::color{ 238, 154, 73 } },
-    std::pair{ "tan3"sv, til::color{ 205, 133, 63 } },
-    std::pair{ "tan4"sv, til::color{ 139, 90, 43 } },
     std::pair{ "teal"sv, til::color{ 0, 128, 128 } },
-    std::pair{ "thistle"sv, til::color{ 216, 191, 216 } },
-    std::pair{ "thistle1"sv, til::color{ 255, 225, 255 } },
-    std::pair{ "thistle2"sv, til::color{ 238, 210, 238 } },
-    std::pair{ "thistle3"sv, til::color{ 205, 181, 205 } },
-    std::pair{ "thistle4"sv, til::color{ 139, 123, 139 } },
-    std::pair{ "tomato"sv, til::color{ 255, 99, 71 } },
-    std::pair{ "tomato1"sv, til::color{ 255, 99, 71 } },
-    std::pair{ "tomato2"sv, til::color{ 238, 92, 66 } },
-    std::pair{ "tomato3"sv, til::color{ 205, 79, 57 } },
-    std::pair{ "tomato4"sv, til::color{ 139, 54, 38 } },
-    std::pair{ "turquoise"sv, til::color{ 64, 224, 208 } },
-    std::pair{ "turquoise1"sv, til::color{ 0, 245, 255 } },
-    std::pair{ "turquoise2"sv, til::color{ 0, 229, 238 } },
-    std::pair{ "turquoise3"sv, til::color{ 0, 197, 205 } },
-    std::pair{ "turquoise4"sv, til::color{ 0, 134, 139 } },
     std::pair{ "violet"sv, til::color{ 238, 130, 238 } },
-    std::pair{ "violetred"sv, til::color{ 208, 32, 144 } },
-    std::pair{ "violetred1"sv, til::color{ 255, 62, 150 } },
-    std::pair{ "violetred2"sv, til::color{ 238, 58, 140 } },
-    std::pair{ "violetred3"sv, til::color{ 205, 50, 120 } },
-    std::pair{ "violetred4"sv, til::color{ 139, 34, 82 } },
     std::pair{ "webgray"sv, til::color{ 128, 128, 128 } },
     std::pair{ "webgreen"sv, til::color{ 0, 128, 0 } },
     std::pair{ "webgrey"sv, til::color{ 128, 128, 128 } },
     std::pair{ "webmaroon"sv, til::color{ 128, 0, 0 } },
     std::pair{ "webpurple"sv, til::color{ 128, 0, 128 } },
-    std::pair{ "wheat"sv, til::color{ 245, 222, 179 } },
-    std::pair{ "wheat1"sv, til::color{ 255, 231, 186 } },
-    std::pair{ "wheat2"sv, til::color{ 238, 216, 174 } },
-    std::pair{ "wheat3"sv, til::color{ 205, 186, 150 } },
-    std::pair{ "wheat4"sv, til::color{ 139, 126, 102 } },
     std::pair{ "white"sv, til::color{ 255, 255, 255 } },
     std::pair{ "whitesmoke"sv, til::color{ 245, 245, 245 } },
     std::pair{ "x11gray"sv, til::color{ 190, 190, 190 } },
     std::pair{ "x11green"sv, til::color{ 0, 255, 0 } },
     std::pair{ "x11grey"sv, til::color{ 190, 190, 190 } },
     std::pair{ "x11maroon"sv, til::color{ 176, 48, 96 } },
     std::pair{ "x11purple"sv, til::color{ 160, 32, 240 } },
-    std::pair{ "yellow"sv, til::color{ 255, 255, 0 } },
-    std::pair{ "yellow1"sv, til::color{ 255, 255, 0 } },
-    std::pair{ "yellow2"sv, til::color{ 238, 238, 0 } },
-    std::pair{ "yellow3"sv, til::color{ 205, 205, 0 } },
-    std::pair{ "yellow4"sv, til::color{ 139, 139, 0 } },
     std::pair{ "yellowgreen"sv, til::color{ 154, 205, 50 } }
 };
 
+static constexpr std::array<til::color, 100> xorgAppGrayColorTable{
+    til::color{ 190, 190, 190 },

```

Comment:
Here's something bizarre. So it's quite obvious that my design uses the first color without variant as the color at index 0. But there is actually a "gray0", and it's `RGB(0, 0, 0)` which is just black !. Between "gray" and "gray0" I choose to leave only "gray". It is technically possible to make them both accessible... but I just don't really understand this.